src/admin/components/forms/field-types/Array/types.ts

```diff
@@ -7,6 +7,7 @@ export type Props = Omit<ArrayField, 'type'> & {
   path?: string
   fieldTypes: FieldTypes
   permissions: FieldPermissions
+  label: string
```

Comment:
This should also be allowed to be set to false as well, right? As the Blocks type?